library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -499,3 +499,5 @@ SAMLGeneral_c=This project is intended to be a set of blackbox tests that verify
 SAMLGeneral_d=An error occurred while logging in.
 
 SAMLGeneral_e=Expected a successful SAML response status code but got a SAML error status code.
+
+SAMLGeneral_f=An error occurred while running the DDF profile tests.
```

Comment:
✏️ What do you think of removing this message and just mentioning that this test is DDF-specific in the test name? Like this https://github.com/connexta/saml-conformance/pull/180/files#diff-2699a8d35a33214ad873f4e892322231R169
In my opinion, the fact that it's DDF-specific should be called out in the test name regardless. To me, it's a little redundant to have it in the test name and in the error message.

Reply:
Oh I see because you're adding a verification that needs a spec ref. Right. Maybe you create a CORE ref message for the snippet that talks about the AuthnContextClass stuff. Even though it's not a MUST, I think that'd be okay. What do you think?